equinox/nn/composed.py

```diff
@@ -160,6 +160,8 @@ class Lambda(Module):
     Precisely, this just adds an extra `key` argument (that is ignored). Given some
     function `fn`, then `Lambda` is essentially a convenience for `lambda x, key: f(x)`.
 
+    If you get a TypeError saying the function is not a valid JAX type, see the FAQ.
+
```

Comment:
1. Can you add a hyperlink to the FAQ entry?

2. I think it might be worth wrapping this into an admonition:
```
!!! faq

    If you get...
```

3. Do you think it would be worth adding the same note in the docstring for `MLP`?